ipynb/chromeos/ChromeOS_Tests.ipynb

```diff
@@ -0,0 +1,827 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Run Tests on ChromeOS"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook shows how to run tests on a Chromebook using `test_that`. At the end of the test results are collected as a dictionary as well as trace events if you specify so in the target configuration.\n",
+    "\n",
+    "**NOTE**: if you want to receive a token from the benchmark before starting trace collection and power measurement, you need to modify the benchmark such that a specific UDP packet is sent to the host machine. As a reference, consider the following python script and convert it to the language in which the benchmark is written accordingly:\n",
+    "\n",
+    "```python\n",
+    "import socket\n",
+    "sock = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)\n",
+    "sock.sendto(\"POWER\", (<TARGET_IP>, 1234))\n",
+    "sock.close()\n",
+    "```\n",
+    "\n",
+    "You should put this right before the instruction that starts the execution of the actual workload in the benchmark."
```

Comment:
Can we add here, as an example, the description of how to apply this patch for WebGL acquarium?
I mean, describe where the file with the benchmark is located (relative to a ChromeOS installation) and in which function it has to be placed?
This can be useful as a reference for ourself too.